var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -173,7 +173,8 @@ class Vtk(CMakePackage):
     depends_on("gl2ps", when="@8.1:")
     depends_on("gl2ps@1.4.1:", when="@9:")
     depends_on("proj@4", when="@8.2.0")
```

Comment:
I know it's not really part of this PR, but can we add the constraint on the lower bound as well? I.e. right now vtk@7 could match to proj 9

Reply:
fixed